git-setup/github-setup-with-access-token.md

```diff
@@ -48,23 +48,52 @@ Then we can copy the token.  **We must copy the token before leaving this page,
 
 ## Using the Token in Git
 
-We can then use the token in git by cloning a repository using HTTPS.
+We can then use the token in git by forking, cloning, and pushing changes to a repository using HTTPS.
 
-We can first go to the [Python Fizbuzz Repository](https://github.com/AdaGold/python-fizzbuzz).
+First go to the [Python Fizbuzz Repository](https://github.com/AdaGold/python-fizzbuzz).
+
+Fork the repository to your GitHub account.
+
+![Fork](../assets/github-setup__fork-repo1.png)
+![Fork Repository](../assets/github-setup__fork-repo2.png)
 
 Then click on the **Code** button and select **https** and copy the link.
 
 ![Clone Repository](../assets/github-setup__clone-repo.png)
 
-Then we can clone the repository using the token.  We should **not** use a password, but rather the copied personal access token.  We can use `cmd-v` to paste the token into the terminal.
+Clone the repository with the following command: 
+
+```
+git clone https://github.com/beccaelenzil/python-fizzbuzz.git
+```
+
+Move your location into the local repository:
```

Comment:
This makes sense and I think it would be fine to commit this change as is, but what do you think about saying this "Now that you have cloned the repository to your own machine, you can move your location into the local repository" or does that seem like overkill?

Reply:
I was thinking how it is follows the same format as the other directions ("Clone the respository..", "Open the directory..."). I am inclined to leave it how it is, but if you really think it's too terse, I could be convinced!

Reply:
Cool, let's leave it!